pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraIdentifiers.scala

```diff
@@ -9,39 +9,100 @@ import uk.ac.wellcome.sierra_adapter.model.SierraBibNumber
 
 // Populate wwork:identifiers.
 //
-//    We populate with three identifier schemes:
+//    We populate with the following identifiers:
 //
-//    -   "sierra-system-number" for the full record type (incl. prefix and
-//        check digit)
 //    -   "sierra-identifier" for the 7-digit internal ID
+//
 //    -   "isbn" from MARC tag 020 subfield $a.  This is repeatable.
 //        https://www.loc.gov/marc/bibliographic/bd020.html
 //
-//    Adding other identifiers is out-of-scope for now.
+//    -   "issn" from MARC tag 022 ǂa.  This is repeatable.
+//        https://www.loc.gov/marc/bibliographic/bd022.html
+//
+//    -   "wellcome-digcode" from MARC tag 759 ǂa.  This is repeatable.
+//        Note: MARC 759 is not assigned by the Library of Congress.
```

Comment:
Note: this is somewhat conjecture on my part. Normally we link to the documentation for a MARC field, but there isn't any for 759: https://www.loc.gov/marc/bibliographic/bd70x75x.html

I don't know if it's worth scratching any deeper on this question right now, or finding out exactly how or why 759 is assigned – but I figured it worth a signpost to say “yeah, don’t be surprised if you can’t find other discussion about 759 on the web”.

Reply:
It’s a local field, it’s not part of the standard